platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -231,48 +239,43 @@ private static CloseableHttpClient getSecureHttpClient(boolean retryRequestsOnEr
   }
 
   private static String[] getProtocols() {
-    if (System.getProperty("https.protocols") != null) {
-      return StringUtils.split(System.getProperty("https.protocols"), ",");
+    if (System.getProperty(HTTPS_PROTOCOLS) != null) {
+      return StringUtils.split(System.getProperty(HTTPS_PROTOCOLS), ",");
     } else {
       return DEFAULT_PROTOCOLS.toArray(new String[DEFAULT_PROTOCOLS.size()]);
     }
   }
 
   private static String[] getCipherSuites() {
-    if (System.getProperty("https.cipherSuites") != null) {
-      return StringUtils.split(System.getProperty("https.cipherSuites"), ",");
+    if (System.getProperty(HTTPS_CIPHER_SUITES) != null) {
+      return StringUtils.split(System.getProperty(HTTPS_CIPHER_SUITES), ",");
     } else {
       return DEFAULT_CIPHER_SUITES.toArray(new String[DEFAULT_CIPHER_SUITES.size()]);
     }
   }
 
   private static SSLContext getSslContext() {
-    if (System.getProperty("javax.net.ssl.keyStore") == null
+    if (System.getProperty(KEY_STORE) == null
         || //
```

Comment:
✏️  Not necessary to change, but we could eventually remove these comments to enforce the styling since our current formatter probably wont make it look atrocious anymore.